boards/beagle/beagleplay_cc1352/beagleplay_cc1352.dts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2022 Jason Kridner, BeagleBoard.org Foundation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <ti/cc1352r7.dtsi>
+#include "beagleplay-cc1352-pinctrl.dtsi"
+
+/ {
+	model = "beagleplay";
+	compatible = "beagle,beagleplay-cc1352";
+
+	aliases {
+		led0 = &led0;
+		led1 = &led1;
+	};
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,flash = &flash0;
+		zephyr,ieee802154 = &ieee802154g;
+	};
+
+	leds: leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio0 27 GPIO_ACTIVE_HIGH>;
+			label = "LED1";
+		};
+
+		led1: led_1 {
+			gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
+			label = "LED2";
+		};
+
+		/* U.FL connector switch */
+		rf_sw: rf_sw {
+			label = "RF_SW";
+			gpios =
+				<&gpio0 29 GPIO_ACTIVE_HIGH>, // SubG TX +20dB
+				<&gpio0 30 GPIO_ACTIVE_HIGH>; // SubG TX/RX 0dB
+		};
```

Comment:
I guess we need some kind of gpio switch bindings in devicetree that behave similarly to the led driver.

@Ayush1325 - to simplify things, group rf_sw inside of its own scope, similarly to the leds, and duplicate the driver.

That would be the easiest solution.

Reply:
Nice catch 😁

Reply:
I have created the PR to fix this for bcf first: https://github.com/zephyrproject-rtos/zephyr/pull/64718 . Will make changes here once bcf stuff is merged.

Reply:
@Ayush1325 looks like to shared the link to *this* PR, not the one you meant.

Reply:
Correct PR to reference is https://github.com/zephyrproject-rtos/zephyr/pull/71403

Reply:
Fixed: https://github.com/zephyrproject-rtos/zephyr/pull/73488